### PR TITLE
minlure: use AnalogIn_IIO

### PR DESCRIPTION
**Pull request by lucasdemarchi**

### CODE REVIEW

**Comment by lucasdemarchi** [comment]
@mirkix could you take a look?

File: libraries/AP_HAL_Linux/AnalogIn_IIO.h [hunk 1]
```diff
@@ -9,11 +9,14 @@
...
 #define IIO_ANALOG_IN_COUNT 8
 #define IIO_ANALOG_IN_DIR "/sys/bus/iio/devices/iio:device0/"
+
 #if CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_PXF
 // Note that echo BB-ADC cape should be loaded
 #define IIO_VOLTAGE_SCALING 0.00142602816
 #elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_BBBMINI
 #define IIO_VOLTAGE_SCALING 3.0*1.8/4095.0
+#elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_MINLURE
+#define IIO_VOLTAGE_SCALING 2.0 / 1000
```
Comment (mirkix) [inline libraries/AP_HAL_Linux/AnalogIn_IIO.h]: could be 2.0 / 1000.0 but does not matter

**Comment by mirkix** [comment]
@lucasdemarchi looks good to me

**Comment by lucasdemarchi** [comment]
@mirkix thanks

Applied.

**Pull request closed without merging**